xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/XMLUtils.java

```diff
@@ -218,12 +218,20 @@ public static String unescapeXMLComment(String content)
     }
 
     /**
-     * Escapes all the XML special characters in a <code>String</code> using numerical XML entities. Specifically,
-     * escapes &lt;, &gt;, ", ', &amp; and {.  Left curly bracket is included here to protect against {{/html}} in
-     * xwiki 2.x syntax.
+     * <p>
+     * Escapes all the XML special characters and a xwiki 2.x special character (i.e., <code>{</code>) in a
```

Comment:
What is a "xwiki 2.x special character"? Maybe you mean "a XWiki Syntax 2.0+ special character"? Now this is already mentioned at the end of the sentence. 

Note: I think `</p>` leads to error in the javadoc generation but I'm not sure. I don't think tags must be closed in javadoc (it's not pure HTML). See for ex https://blog.joda.org/2012/11/javadoc-coding-standards.html

I think the javadoc still missing explanation as to why you would use this method. See my proposals on the chat.

Reply:
Javadoc update to use better writing standard + adding more explanation on why each method should be used for